### ISSUE TITLE
3.5 Examples?

### ISSUE THREAD

**Issue by avallintine** [pixijs/pixi-spine#152]
Where can I find an example project exported with 3.5, so I can test pixi-spine for my purposes? All the examples included with the trial version of Spine seem to be 3.4.02, and the trial won't allow me to export. Thanks!

**Comment by ivanpopelyshev** [comment]
According to license, you cant use pixi-spine at all :( 

I'll add them in pixi-spine this week , is that ok? :)

**Comment by avallintine** [comment]
Sure, that would be awesome. I'm aware of the licensing restrictions on using runtimes, but considering this:

http://esotericsoftware.com/spine-download

> The trial comes with many example projects which include exported JSON animation data and texture atlases. These can be used to evaluate how the Spine Runtimes work with your game toolkit before purchasing.

I can't test the runtime without current examples :)

**Comment by ivanpopelyshev** [comment]
We'll add same examples as in spine this week. Do you want to test something specific? Like, embedding stuff into webpack, or using custom atlases?

**Comment by avallintine** [comment]
Nothing in particular, just wanted to get a feel for the API by interacting with some working animations. If you update the example files included with the Spine trial to 3.5 that would probably suit me just fine.

**Comment by ivanpopelyshev** [comment]
Ok, in a few hours then

**Comment by avallintine** [comment]
Excellent, much appreciated.

**Comment by ivanpopelyshev** [comment]
Actually, you can take them from official runtimes: https://github.com/EsotericSoftware/spine-runtimes/tree/master/spine-ts/webgl

It also contains code for webgl, and I'm gonna port that demo to pixi. But if you want to test it right now, just take those files and port it for yourself ^_^

**Comment by avallintine** [comment]
Works for me. Thanks!

**Comment by mspanish** [comment]
I'd love a sample of using the scaling via textures & atlases! Spine exports different resolutions into folders: so I have 0.25, 0.5, and 1 folders that were Spine generated. Inside of them, are just crab.png and crab.atlas - at the varying resolutions. Per the docs it looks like I need to rename these, right? So I'd have to move them all into t common folder, and rename them something like crab.@2x.png and crab.@2x.atlas?  Is that right?

**Comment by ivanpopelyshev** [comment]
@mspanish 

Isnt this one enough? 

https://github.com/pixijs/pixi-spine/tree/master/examples

**Comment by ivanpopelyshev** [comment]
Yes, you have to rename it if you want to use auto-choose feature.

**Comment by ivanpopelyshev** [comment]
@mspanish 

auto-choose works only if you add pixi-compressed-textures. 

But even if you dont use it, you have to name files properly. I thought spine exports multiple resolutions atlas with those suffixes by default?

**Comment by cursedcoder** [comment]
@ivanpopelyshev not really, it's a configuration matter
![screen shot 2017-02-13 at 00 28 51](https://cloud.githubusercontent.com/assets/899452/22863927/6c9b778c-f183-11e6-93d8-5a17099fcf4f.png)

**Comment by mspanish** [comment]
yes that example was fine - it was operator error :) I was trying to scale -and your other code worked perfectly, and code for scaling. THANK YOU!
